src/Chainweb/Chainweb.hs

```diff
@@ -884,7 +875,7 @@ runChainweb cw = do
             FastTimedCPM{} -> enabled c
             Development -> enabled c
             Testnet03 -> enabled c
-            Mainnet01 -> disabled
+            Mainnet01 -> enabled c
```

Comment:
Mempools are now completely on, but TXs are still blocked (for the mempool) in three places:

1. `/mempool/insert`
2. The pre-insert check
3. Block Validation

Do you foresee any issues with thus @gregorycollins ?

Reply:
A node of mine has been running with just these mempool changes for several hours without issue.